### ISSUE TITLE
Invalid argument supplied for foreach() - displayMetaTags() error on freshly created model instances

### ISSUE THREAD

**Issue by tonyneurony** [VarboxInternational/varbox#19]
- Laravel Version: 8.0
- Varbox Version: 2.1
- PHP Version: 7.3

### Description of the bug (screenshots might help):
When you add a HasMetaTags trait to a model, that previously had some entries in DB (for example i seed pages for Page model), `meta` column remains `null` in DB, until you save that model in DB(so it saves the json data to meta column), you will be getting 'Invalid argument supplied for foreach()' error.
The fix is quick:
in `displayMetaTags()` method, wrap the foreach into
`if (!empty($tags)) {`

**Comment by zbiller** [comment]
Hi @tonyneurony 

Can you please let me know if you casted the `meta` column to an `array` as per the [docs](https://varbox.io/docs/2.x/meta-tags#apply-the-trait)?

```
/**
 * The attributes that should be casted to native types.
 *
 * @var array
 */
protected $casts = [
    ...
    'meta' => 'array',
];
```

**Comment by tonyneurony** [comment]
Hi @zbiller 
Yes, for sure. This actually happens on default Page model as well, when, for example u have a page seeded via seeder, thus the meta columns remains 'null' in DB

**Comment by zbiller** [comment]
@tonyneurony 

I've applied a fix for this issue.
It's available on version `2.2.0`.

Thanks for your support!
